src/tools/rbd/action/Bench.cc

```diff
@@ -293,18 +332,18 @@ int execute(const po::variables_map &vm) {
     return r;
   }
 
-  r = do_bench_write(image, bench_io_size, bench_io_threads, bench_bytes,
-                     bench_random);
+  r = do_bench(image, bench_io_type, bench_io_size, bench_io_threads,
+		     bench_bytes, bench_random);
   if (r < 0) {
-    std::cerr << "bench-write failed: " << cpp_strerror(r) << std::endl;
+    std::cerr << "do_bench failed: " << cpp_strerror(r) << std::endl;
     return r;
   }
   return 0;
 }
 
 Shell::Action action(
-  {"bench-write"}, {}, "Simple write benchmark.", "", &get_arguments, &execute);
+  {"bench"}, {}, "Simple benchmark.", "", &get_arguments, &execute);
```

Comment:
thanx @dillaman that's my first version of this feature, but I found there was lots of redundancy. But the compatibility is important for any project. Then I will add a bench-read rather than rename bench-write to bench. And refactor them to reduce the redundancy.